Userland/DevTools/HackStudio/Dialogs/ProjectTemplatesModel.h

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2021, Nick Vella <nick@nxk.io>
+ * Copyright (c) 2021, the SerenityOS developers.
```

Comment:
Touching a file to only modify the copyright header? :^)

Reply:
Oops, I was adding the copyright headers at the last minute. :^)